<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedBindingEditorDialog</class>
 <widget class="QDialog" name="AdvancedBindingEditorDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>401</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>221</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="Btn_Ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>951</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>32</number>
    </property>
    <item>
     <widget class="QBindingExchangeWidget" name="Exch_GamepadButtons" native="true"/>
    </item>
    <item>
     <widget class="QBindingExchangeWidget" name="Exch_InputActionKeys" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QBindingExchangeWidget</class>
   <extends>QWidget</extends>
   <header>QBindingExchangeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Ok</sender>
   <signal>clicked()</signal>
   <receiver>AdvancedBindingEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>AdvancedBindingEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
